pkg/scheduler/framework/v1alpha1/cycle_state.go

```diff
@@ -44,31 +44,50 @@ type StateKey string
 type CycleState struct {
 	mx      sync.RWMutex
 	storage map[StateKey]StateData
-	// if recordFrameworkMetrics is true, framework metrics will be recorded for this cycle.
-	recordFrameworkMetrics bool
+	// if recordExtensionPointMetrics is true, FrameworkExtensionPointDuration will be recorded for this cycle.
+	recordExtensionPointMetrics bool
```

Comment:
This is not needed anymore since we always record `FrameworkExtensionPointDuration`.

Reply:
Please address this. I will LGTM after this is addressed.